modules/mtl/arrow-mtl-data/src/test/kotlin/arrow/mtl/OptionTTest.kt

```diff
@@ -145,3 +136,21 @@ class OptionTTest : UnitSpec() {
     }
   }
 }
+
+fun <F> OptionT.Companion.genk(genkF: GenK<F>) = object : GenK<Kind<ForOptionT, F>> {
+  override fun <A> genK(gen: Gen<A>): Gen<Kind<Kind<ForOptionT, F>, A>> = genkF.genK(Gen.option(gen)).map {
+    OptionT(it)
+  }
+}
+
+fun IO.Companion.eqK(timeout: Duration = 60.seconds) = object : EqK<ForIO> {
+  override fun <A> Kind<ForIO, A>.eqK(other: Kind<ForIO, A>, EQ: Eq<A>): Boolean =
+    (this.fix() to other.fix()).let {
+      val ls = it.first.attempt().unsafeRunTimed(timeout)
+      val rs = it.second.attempt().unsafeRunTimed(timeout)
+
+      Option.eq(Either.eq(Eq.any(), EQ)).run {
+        ls.eqv(rs)
+      }
+    }
+}
```

Comment:
This is needed in a few places I think, can we place this somewhere general? It needs to stay in the tests, but keeping it in a specific test sucks. If there is no general file just place it where the IO tests are and import it from there

Reply:
don't think it can go directly to the IO tests (if you mean IOTest) as this is not visible from the mtl module. i put it in an extra extension file next to the tests.